### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by techlover1**
Pending a review both by myself and @mikemaddem. I also need to test a full setup from scratch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,64 @@
...
+3. Configure settings.py
+4. Apply database migrations by running `python3 manage.py migrate` (assuming your database options are set as you'd like in settings.py)
+5. Create a django admin/super user account - `python manage.py createsuperuser` and follow the prompts
+6. Give your new superuser account staff panel access by modifying the database row in the `profiles_userprofile` table - set
+`user_verified` to `1` and set `user_type` to `superadmin` or `admin` (hint: you can use django admin at /admin)
+7. Create a StaticInfo object, if you don't every page will throw an error telling you that it can't access the StaticInfo - easiest
 way to make a staticinfo object is through the django admin at /admin
+8. Setup a web server and reverse proxy using industry standard best practices (hint: gunicorn and nginx work well)
 
-# Docker Instalation
```
Comment (mikemaddem) [inline README.md]: nice job here buddy

File: README.md [hunk 2]
```diff
@@ -1,30 +1,64 @@
...
 
-# Configuration
-You can and probably should configure things to however you'd like! As a general project we try to make things as flexible as possible
-but sometimes we just can't have it do everything in the world :(
+1. Clone the repository! You need the code, it won't just magically work without it. (Seems to be a recurring theme)
+2. Create the folder for static files by running `sudo mkdir /var/www/static` and transfer ownership by running `sudo chown www-data:www-data /var/www/static`. Feel free to substitute the username, but your reverse proxy must be running as that user
+3. Run `docker build .` (take note of the image id) followed by `docker run -d -v /var/www/static:/static -v /tmp:/sock --net=host -u www-data --name project-olly [your image id]`
+4. Setup a reverse proxy of your choice (we highly recommend nginx, and have setup instructions	available in [nginx_setup.md](nginx_setup.md))
+5. A user was automatically created with the username `admin` and the password `ChangeMe!` **DON'T FORGET TO CHANGE THE PASSWORD!**
+6. How easy was that!?
```
Comment (mikemaddem) [inline README.md]: not easy enough. bad

File: README.md (hunk 1 above)
Comment (techlover1) [reply README.md]: Fixed in 98ce8e53ad699b313fc7b3b65f76d18790d83b4c

File: README.md (hunk 2 above)
Comment (techlover1) [reply README.md]: ¯\_(ツ)_/¯

File: README.md [hunk 3]
```diff
@@ -1,30 +1,64 @@
-# project-olly
+# Project Olly
+[![Build Status](https://travis-ci.org/NFM-Studios/project-olly.svg?branch=master)](https://travis-ci.org/NFM-Studios/project-olly)
 [![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors)
+<a href="https://discord.gg/5dp8x2t">
+    <img src="https://img.shields.io/badge/discord-join-7289DA.svg?logo=discord&longCache=true&style=flat" />
+</a>
+
+## Please note: this document is currently under construction and is not yet complete.
```
[the remark this answers is not in the document]
Comment (techlover1) [reply README.md]: Probably should

[the remark this answers is not in the document]
Comment (mikemaddem) [reply README.md]: fixed

**Pull request merged**